user/tutorials/User Guide for Translation Tool/User Guide for the Translation Tool.md

```diff
@@ -0,0 +1,48 @@
+
+
+| title                               | category | description                             |
```

Comment:
To work with GitHub pages we need this in yaml format.
```
---
title: ...
category: ... 
description: ...
---
```

Reply:
Do you mean put this in the header?

Reply:
Yes place this at the top of the file.